### PR TITLE
Final Preparations for 2.0 Compatibility

### PR DESCRIPTION
**Pull request by CodaFi**
Just waiting on a release from Focus at this point.

### CODE REVIEW

File: Aquifer.xcodeproj/xcshareddata/xcschemes/Aquifer.xcscheme [hunk 1]
```diff
@@ -3,7 +3,7 @@
    LastUpgradeVersion = "0700"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "YES"
+      parallelizeBuildables = "NO"
```
Comment (pthariensflame) [inline Aquifer.xcodeproj/xcshareddata/xcschemes/Aquifer.xcscheme]: Why this change?

Comment (CodaFi) [inline Aquifer.xcodeproj/xcshareddata/xcschemes/Aquifer.xcscheme]: Not sure.  Xcode's been doing that to all the schemes of late.  I'll try to change it back.

**Comment by pthariensflame** [comment]
Ok.  Just waiting on dependencies, then!

**Comment by CodaFi** [comment]
Do you want to release Focus or should I?

**Comment by pthariensflame** [comment]
You can, since you've done it before.  I basically just do Aquifer right now, although if you'd like that to change it totally can.  :)

**Comment by CodaFi** [comment]
Welp, that does it.

:boat:

**Comment by pthariensflame** [comment]
Yay!
